docs/core/install/linux-package-manager-debian9.md

```diff
@@ -69,3 +69,11 @@ sudo apt-get install dotnet-runtime-3.1
 ## How to install other versions
 
 [!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
+
+## Troubleshoot the package manager
+
+This section provides information on common errors you may get while using the package manager to install .NET Core.
+
+### Failed to fetch
+
+While installing the .NET Core package, you may see an error similar to `Failed to fetch ... File has unexpected size ... Mirror sync in progress?`. Generally speaking, this error means that the package feed for .NET Core is being upgraded with newer package versions, and that you should try again later. During an upgrade, the package feed should not be unavailable for more than 2 hours. If you continually receive this error for more than 2 hours, please file an issue at <https://github.com/dotnet/core/issues>.
```

Comment:
30 minutes for the Debian package repos (includes Debian and Ubuntu). The 2 hours is for RPM feeds like Fedora, openSUSE, SLES, and CentOS. (Not RHEL because those use Red Hat repos, not Microsoft ones.)

Reply:
Whoops. See, I don't fully understand this stuff yet. 👎  I'll fix that